components/Card.js

```diff
@@ -84,4 +84,49 @@ CardAddress.propTypes = {
   label: PropTypes.string,
 };
 
+export function PositionRequirements({ label, children }) {
+  return (
+    <div className="p-2 shaded-text" style={{ marginTop: 'auto' }}>
```

Comment:
could we achieve this `style={{ marginTop: 'auto' }}` with tailwind and not to use inline styles?

Reply:
Definitely!